testing/trino-product-tests/src/main/java/io/trino/tests/product/utils/QueryExecutors.java

```diff
@@ -25,27 +25,27 @@
 
 public final class QueryExecutors
 {
-    @Deprecated
-    public static QueryExecutor onPresto()
-    {
-        return onTrino();
-    }
-
     public static QueryExecutor onTrino()
     {
-        return connectToPresto("presto");
+        return connectToTrino("presto");
     }
 
     public static QueryExecutor onCompatibilityTestServer()
     {
-        return connectToPresto("compatibility-test-server");
+        return connectToTrino("compatibility-test-server");
```

Comment:
Notably this server can be both Trino or Presto according to our test matrix.
Not sure if it matters though.